custom-stylelint-plugins/no-missing-var.js

```diff
@@ -0,0 +1,23 @@
+const stylelint = require('stylelint');
+
+const ruleName = 'custom-rule/no-missing-var';
+
+const plugin = stylelint.createPlugin(ruleName, (enabled, options) => {
```

Comment:
Same as above, this should be covered by https://stylelint.io/user-guide/rules/custom-property-no-missing-var-function

Reply:
This rule only reports custom properties that are defined within the same source.

Will not throw error here:
.spectrum-Well {
  background-color: (--spectrum-well-background-color);
  border-color: var(--spectrum-well-border-color);
}


But will only throw error here where the properties are defined on the same css source.
    :root { --foo: red; }
    a { color: --foo; }